.github/workflows/update.yml

```diff
@@ -0,0 +1,32 @@
+name: Update flake inputs
+on:
+  schedule:
+    # Update every Sunday and Wednesday
+    - cron: "51 3 * * 0,3"
+  workflow_dispatch:
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+        persist-credentials: false
+        fetch-depth: 0
+      - uses: cachix/install-nix-action@v13
+        with:
+          install_url: https://github.com/numtide/nix-unstable-installer/releases/download/nix-2.4pre20210415_76980a1/install
+          extra_nix_config: |
+            experimental-features = nix-command flakes
+      - name: Update & commit lockfile
+        run: |
+          git config --local user.email "flakebot@users.noreply.github.com"
+          git config --local user.name "flakebot"
+          nix flake update --commit-lock-file
```

Comment:
Might be a good idea to verify that `nix run github:nix-community/home-manager` will run with exit code 0 before proceeding with the push action

Reply:
Good point, I can add that as an extra step.

Reply:
Did you end up adding this? Can't find it anywhere